web/src/components/CategoryListItem.tsx

```diff
@@ -65,25 +42,39 @@ type CategoryListItemProps = {
 }
 
 const CategoryListItem = ({ category, subCategories }: CategoryListItemProps): ReactElement => {
-  const SubCategories = subCategories.map(subCategory => (
-    <SubCategory key={subCategory.path} dir='auto'>
-      <StyledLink to={subCategory.path}>
-        {!!subCategory.thumbnail && <CategoryThumbnail alt='' src={subCategory.thumbnail} />}
-        <SubCategoryCaption>{subCategory.title}</SubCategoryCaption>
-      </StyledLink>
-      <Divider />
-    </SubCategory>
-  ))
+  const SubCategories = subCategories.map(subCategory => {
+    const { path, thumbnail, title } = subCategory
+    return (
+      <SubCategoryListItem key={path}>
+        <StyledSubCategoryListItemButton component={Link} to={path}>
+          {!!thumbnail && (
+            <ListItemAvatar>
+              <Avatar src={thumbnail} variant='square' />
+            </ListItemAvatar>
+          )}
```

Comment:
🔧 As discussed with @hauf-toni and @laravoelk: We should remove 3rd level icons here.
```suggestion
```